schemas/session-replay-mobile-events-schema.json

```diff
@@ -6,10 +6,16 @@
   "description": "Schema of all properties of a mobile Session Replay event",
```

Comment:
**nitpic:** you can remove the `Mobile` since it's now records & segments that are shared across platform

Reply:
My idea was that once you will add your own schemas here we will have a `session-replay-browser-events-schema.json` also